iron-localstorage.html

```diff
@@ -179,7 +179,12 @@
      * @param {boolean=} externalChange true if loading changes from a different window
      */
     _load: function(externalChange) {
-      var v = window.localStorage.getItem(this.name);
+      try {
+        var v = window.localStorage.getItem(this.name);
+      } catch (ex) {
+        this.errorMessage = ex.message;
+        this._error("LocalStorage could not be saved. Safari cookies always blocked?", ex);
```

Comment:
Actually, could we make this message a little nicer? Something like "Could not save to localStorage. Please ensure cookies are not blocked on this page".

It could be non-Safari browsers doing this too, no?